rascal2/ui/view.py

```diff
@@ -232,4 +240,13 @@ def init_settings_and_log(self, save_path: str):
             log_path = proj_path / log_path
 
         log_path.parents[0].mkdir(parents=True, exist_ok=True)
-        self.logging = setup_logging(log_path, level=self.settings.log_level)
+        self.logging = setup_logging(log_path, self.terminal_widget, level=self.settings.log_level)
+
+    def handle_results(self, results):
+        """Handle the results of a RAT run."""
+        self.end_run()
+        self.controls_widget.chi_squared.setText(f"{results.calculationResults.sumChi:.6g}")
+
+    def end_run(self):
```

Comment:
I recommend changing this function name to better reflect what it does i.e. reset_xxx instead of end_run